### ISSUE TITLE
spotbugsMain task in Gradle throws ExceptionInInitializerError

### ISSUE THREAD

**Issue by sunnypatel2141** [spotbugs/spotbugs-gradle-plugin#358]
Our build runs `./gradlew --info spotbugsMain` task. 

We're using `toolVersion = '4.1.3'`

This is throwing the following error in one of our subprojects:
```
java.lang.ExceptionInInitializerError
	at java.base/java.lang.J9VMInternals.ensureError(J9VMInternals.java:186)
	at java.base/java.lang.J9VMInternals.recordInitializationFailure(J9VMInternals.java:175)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:59)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:73)
Caused by: java.lang.NullPointerException
	at worker.org.gradle.internal.classloader.FilteringClassLoader.<clinit>(FilteringClassLoader.java:51)
	... 2 more
```

Then later on, another subproject fails with 
```
* What went wrong:
Execution failed for task ':extension-relational-extent:spotbugsMain'.
> A failure occurred while executing com.github.spotbugs.snom.internal.SpotBugsRunnerForWorker$SpotBugsExecutor
   > Failed to run Gradle Worker Daemon
      > Process 'Gradle Worker Daemon 98' finished with non-zero exit value 1
```

Is the issue the number of Gradle Worker Daemon's ?

Have you seen this before ?

Thanks,
Sunny

**Comment by davidobber** [comment]
I have fixed the same error just today, and in my case there was a transitive dependency in my project with groovy-all-2.5.13, while my project included a 2.5.+ version that was different (2.5.14). I just excluded the 2.5.13 from the transitive deps and I fixed it. The --info or --debug flag output told me that the problem was in groovy-datetime 2.5.13 vs 2.5.14. 
Hope it helps.

**Comment by KengoTODA** [comment]
Please share [mcve project](https://stackoverflow.com/help/minimal-reproducible-example) to reproduce the issue in our local.
